src/components/SignInOptions.tsx

```diff
@@ -0,0 +1,176 @@
+import React from "react"
+import { ElementAppearance } from "../AppearanceProvider"
+import { Button, ButtonProps } from "../elements/Button"
+import { Config } from "../useConfig"
+
+export type SignInOptionsProps = {
+    config: Config | null
+    buttonAppearance?: ElementAppearance<ButtonProps>
+}
+
+export const SignInOptions = ({ config, buttonAppearance }: SignInOptionsProps) => {
+    const GOOGLE_LOGIN_PATH = "/google/login"
+    const GITHUB_LOGIN_PATH = "/github/login"
+    const SLACK_LOGIN_PATH = "/slack/login"
+    const MICROSOFT_LOGIN_PATH = "/microsoft/login"
+    const LINKEDIN_LOGIN_PATH = "/linkedin/login"
+    const PASSWORDLESS_LOGIN_PATH = "/login_passwordless"
+
+    const loginWithSocial = (path: string) => {
+        const url = path // use url from config?
```

Comment:
Yeah, for loginWithSocial this should be the authUrl followed by the paths above (Google => LinkedIn). 

For passwordless, given that that's a component now, we'll need to load that from the config.

Reply:
[fix](https://github.com/PropelAuth/react/pull/11/commits#:~:text=handle%20loading%20%26%20error%20state%20for%20mfa)